bliss/conf/base_config.yaml

```diff
@@ -81,7 +81,9 @@ encoder:
     input_transform_params:
         use_deconv_channel: false
         concat_psf_params: false
-        log_transform: true
+        log_transform:
+            use_log_transform: true
```

Comment:
Can we omit the `use_log_transform` boolean here? Instead, always use the log transform if it's provided, and don't if it's not.

Reply:
Had to make a few other changes so that the tests would all pass with the current baseline model - a few might still fail  (i.e. `test_sdss_reconstruct`, `test_simulate`) because I did the log transform a bit differently for the baseline model than how it is currently done in the encoder. But everything should still run fine.

I have to upload a new model anyways prior to the version release - is this fine?

Reply:
Sounds good